plugins/modules/cargo.py

```diff
@@ -68,6 +68,15 @@
     type: path
     required: false
     version_added: 9.1.0
+  features:
+    description:
+      - List of features to activate.
+      - This is only used when installing packages.
+    type: list
+    elements: str
+    required: false
+    default: []
+    version_added: 10.8.0
```

Comment:
Still I'm not sure about it, you can easily specify it later and replace with version you want.

Reply:
Until a couple of days ago I would point that this should be `11.0.0` , but now, as ansible-core is going to take longer to be released, I have not yet seen an update on the release of the collection.

@felixfontein any news on that front? TIA

Reply:
Changed to `11.0.0`.